server/routes/analyticsRouter.ts

```diff
@@ -235,54 +235,6 @@ export default function routes(router: Router): Router {
     })
   })
 
-  const protectedCharacteristicsGraphIds = [
-    'incentive-levels-by-ethnicity',
-    'incentive-levels-by-age',
-    'incentive-levels-by-religion',
-    'incentive-levels-by-disability',
-    'incentive-levels-by-sexual-orientation',
-  ]
-  routeWithFeedback('/protected-characteristics', protectedCharacteristicsGraphIds, async (req, res, next) => {
-    res.locals.breadcrumbs.addItems({ text: 'Protected characteristics' })
-
-    // Redirect to new Protected characteristics page when feature flag is on
-    if (config.featureFlags.showAnalyticsPcDropdown) {
-      res.redirect('/analytics/protected-characteristic?characteristic=age')
```

Comment:
i would leave a version of this route in case that just redirects to the new one

Reply:
Good point - addressed